metrics/notifier.go

```diff
@@ -2,26 +2,30 @@ package metrics
 
 // NotifierMetrics is a collection of metrics used in notifier
 type NotifierMetrics struct {
-	SubsMalformed          Meter
-	EventsReceived         Meter
-	EventsMalformed        Meter
-	EventsProcessingFailed Meter
-	EventsByState          MetersCollection
-	SendingFailed          Meter
-	SendersOkMetrics       MetersCollection
-	SendersFailedMetrics   MetersCollection
+	SubsMalformed                  Meter
+	EventsReceived                 Meter
+	EventsMalformed                Meter
+	EventsProcessingFailed         Meter
+	EventsByState                  MetersCollection
+	SendingFailed                  Meter
+	SendersOkMetrics               MetersCollection
+	SendersFailedMetrics           MetersCollection
+	PlotsBuildDurationMs           Meter
+	PlotsEvaluateTriggerDurationMs Meter
 }
 
 // ConfigureNotifierMetrics is notifier metrics configurator
 func ConfigureNotifierMetrics(registry Registry, prefix string) *NotifierMetrics {
 	return &NotifierMetrics{
-		SubsMalformed:          registry.NewMeter("subs", "malformed"),
-		EventsReceived:         registry.NewMeter("events", "received"),
-		EventsMalformed:        registry.NewMeter("events", "malformed"),
-		EventsProcessingFailed: registry.NewMeter("events", "failed"),
-		EventsByState:          NewMetersCollection(registry),
-		SendingFailed:          registry.NewMeter("sending", "failed"),
-		SendersOkMetrics:       NewMetersCollection(registry),
-		SendersFailedMetrics:   NewMetersCollection(registry),
+		SubsMalformed:                  registry.NewMeter("subs", "malformed"),
+		EventsReceived:                 registry.NewMeter("events", "received"),
+		EventsMalformed:                registry.NewMeter("events", "malformed"),
+		EventsProcessingFailed:         registry.NewMeter("events", "failed"),
+		EventsByState:                  NewMetersCollection(registry),
+		SendingFailed:                  registry.NewMeter("sending", "failed"),
+		SendersOkMetrics:               NewMetersCollection(registry),
+		SendersFailedMetrics:           NewMetersCollection(registry),
+		PlotsBuildDurationMs:           registry.NewMeter("plots", "build_duration_ms"),
```

Comment:
Maybe we should write words as separate strings here?

```suggestion
		PlotsBuildDurationMs:           registry.NewMeter("plots", "build", "duration", "ms"),
```

Reply:
fixed